src/host/settings.cpp

```diff
@@ -45,6 +45,7 @@ Settings::Settings() :
     _fCtrlKeyShortcutsDisabled(false),
     _bWindowAlpha(BYTE_MAX), // 255 alpha = opaque. 0 = transparent.
     _fFilterOnPaste(false),
+    _LaunchFaceName{},

```

Comment:
Capital L Launch?

Reply:
preexisting art. happy to change it, but all of the settings in that block are capital.

Reply:
its fine.

Reply:
Oh, I know what this was. I think that underscore then lowercase letter is technically reserved/invalid or something per the standards (even though we use it everywhere) and this particular var was added to-standard. Whatever. It's fine.

Reply:
I think underscore _upper_ is invalid, honestly